packages/FormElement/stories/examples/Inline.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Rule } from "storybook/assets/styles/common.styles";
+import L10n from "@paprika/l10n";
+import Input from "@paprika/input";
+import Button from "@paprika/button";
+import FormElement from "../../src";
+
+const { Layout, Label, Instructions, Content, Description, Error } = FormElement;
+const { LeftCol, RightCol } = Layout;
+
+export default function InlineContent() {
+  const [value, setValue] = React.useState();
+  const [errorText, setErrorText] = React.useState();
+
+  return (
+    <L10n>
+      <FormElement>
+        <Layout>
+          <LeftCol width={180}>
+            <Label>Form Label</Label>
+          </LeftCol>
+          <RightCol>
+            <Instructions>These are some field instructions.</Instructions>
+            <Content>
+              {a11yProps => (
+                <Input
```

Comment:
I still think worth showing multiple inputs inline here

Reply:
I saved that for the `Everything Bagel` story.  I wanted to keep this one streamlined though, so the focus is on the `<Layout>`.